osh/worker/csmock_runner.py

```diff
@@ -119,8 +119,6 @@ def analyze(self, analyzers, srpm_path, profile=None, su_user=None, additional_a
 
         if profile == "cspodman":
             cmd = "cspodman"
-            # XXX: this destroys all containers we have access to, related or not
-            cmd += " --force-global-cleanup-on-exit"
```

Comment:
If we don't need to use it, can we remove all of the code, such as:
https://gitlab.cee.redhat.com/osh/cspodman/-/blame/main/py/cspodman#L62
https://github.com/csutils/csdiff/blob/main/tests/csgrep/0117-csgrep-set-imp-level-stdin.txt#L19
https://github.com/csutils/csdiff/blob/main/tests/csgrep/0117-csgrep-set-imp-level-stdout.txt#L19

Reply:
The option has existed in `cspodman` since 2020.  I think it should go away through a deprecation process.  The code that changed behavior of `cspodman` has already been removed.  Removing the warning right now is not going to help.  It would only break OSH if the changes were applied in wrong order.

The files in `csgrep` are test coverage taken from old scans.  I do not think we need to change scan properties retroactively.

Reply:
I agree, there is no reason to remove it from test files.